message.d.ts

```diff
@@ -11,6 +11,8 @@ export namespace Message {
     from?: User;
     /** Sender of the message, sent on behalf of a chat. For example, the channel itself for channel posts, the supergroup itself for messages from anonymous group administrators, the linked channel for messages automatically forwarded to the discussion group. For backward compatibility, the field from contains a fake sender user in non-channel chats, if the message was sent on behalf of a chat. */
     sender_chat?: Chat;
+    /** text the message was sent */
+    text: string;
```

Comment:
This property is already declared at https://github.com/KnorpelSenf/typegram/blob/8d62616a62b4b402e23225afb225cf8aa4e76f69/message.d.ts#L49